From 3f6a69cb78deb161abc5cbb2def85e2f964f2c85 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Wed, 6 Aug 2014 03:47:36 -0400
Subject: [arm] pci: Update outdated comment for pcibios_bus_report_status()

Message-id: <20140806034736.1438.9118.stgit@gir.stowe>
Patchwork-id: 87442
O-Subject: [RHEL7.1 PATCH 33/69] PCI: Update outdated comment for pcibios_bus_report_status()
Bugzilla: 1110896
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110896
Upstream Status: 5ee09efe5406ad983e98d07ac17ad27819c0dc5b

commit 5ee09efe5406ad983e98d07ac17ad27819c0dc5b
Author: Yijing Wang <wangyijing@huawei.com>
Date:   Thu Feb 13 19:57:43 2014 +0800

    PCI: Update outdated comment for pcibios_bus_report_status()

    pci_find_device() has been superseded by pci_get_device().

    Signed-off-by: Yijing Wang <wangyijing@huawei.com>
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/arm/kernel/bios32.c b/arch/arm/kernel/bios32.c
index b2ed73c..5f232fb 100644
--- a/arch/arm/kernel/bios32.c
+++ b/arch/arm/kernel/bios32.c
@@ -19,7 +19,7 @@
 static int debug_pci;
 
 /*
- * We can't use pci_find_device() here since we are
+ * We can't use pci_get_device() here since we are
  * called from interrupt context.
  */
 static void pcibios_bus_report_status(struct pci_bus *bus, u_int status_mask, int warn)
-- 
1.7.1